.circleci/config.yml

```diff
@@ -363,7 +363,7 @@ jobs:
           command: |
             export UTILS_DOCKERHUB_REPO=mozilla/sync-spanner-py-utils
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              DOCKER_TAG="${CIRCLE_SHA1}"
+              DOCKER_TAG="latest"
```

Comment:
could,should we make the tags "latest,${CIRCLE_SHA1}" ?

Reply:
(I don't know if that's a valid syntax...)

Reply:
It's not, no. Okay - well, looks like the SHA tag is only used to trigger dev builds, which I don't think is a thing we're using or even looking at, so I'd say we could switch back to latest.